tests/contrib/operators/test_kubernetes_job_operator.py

```diff
@@ -0,0 +1,116 @@
+from airflow.models import DAG, TaskInstance
+from datetime import datetime, timedelta
+from airflow.utils.kubernetes_utils import KubernetesContainerInformation
+import airflow.contrib.operators.kubernetes_operator as kubernetes_operator
+import mock
+import unittest
+
+
+class KubernetesJobOperatorTest(unittest.TestCase):
+
+    ARGS = {
+        'owner': 'jessica'
+    }
+    DAG = DAG(
+        dag_id='DAG_ID',
+        default_args=ARGS,
+        schedule_interval="17 1 * * *",
+        start_date=datetime(2019, 10, 1),
+        catchup=False
+    )
+    TASK_ID = 'test-task-id'
+    YAML = """
+        apiVersion: batch/v1
+        kind: Job
+        metadata:
+          name: {{ job_name }}
+        spec:
+          template:
+            spec:
+              containers:
+              - name: test-container-name
+                image: test-container-image-information
+                command: ['test', 'command']
+              restartPolicy: Never
+          backoffLimit: 0
+    """
+    CONTAINER = KubernetesContainerInformation(name='test-container-name',
+                                               image='test-container-image-information',
+                                               command=['test', 'command'])
+
+    SUCCESS_SIDE_EFFECTS = ['applied',
```

Comment:
I think `subprocess.check_output` returns a byte string, not a list. You may want to add `'\n'.join(...)` to this, or in the test function somewhere, to get it to match

Reply:
So the side effects will return one item of the list each time the function is called, not the entire list at once.

But I probably need to update the individual strings to byte strings once I figure out what those are.